src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -1341,6 +1355,10 @@ int sd_dhcp_server_start(sd_dhcp_server *server) {
                         goto on_error;
         }
 
+        r = dhcp_server_load_leases(server);
+        if (r < 0)
+                log_dhcp_server(server, "Failed to load lease file %s, ignoring: %m", strna(server->lease_file));
```

Comment:
btw, if we are offline for a long time during reboot, we might need to immediately terminate these leases? is there code in place for that already?